### DISCUSSION TITLE
Get struct data from C++

### DISCUSSION THREAD

**Discussion by hyf4053** [nxrighthere/UnrealCLR#208] [Help] [answered]
Let's say I defined a struct in C++, it is possible to get my custom struct value and pass the value to my .Net library?

**Comment by nxrighthere** [comment, marked as answer]
This feature is not supported at the moment. Please, check the manual for options that are currently available for data passing.

**Reply by hyf4053** [reply]
Thanks for answering this question. However, there is another question that I want to ask is if I just want to use my .net library in Unreal C++,  is there a better solution to achieve that? I can program in C++ but there is some function in my managed .net library I want to use.

**Reply by nxrighthere** [reply]
Since you are familiar with C++, you can try to use [shared memory](https://www.google.com/search?q=C%2B%2B+C%23+shared+memory) or [inter-process communication](https://www.google.com/search?q=c%2B%2B+c%23+interprocess+communication).
